src/backend.c

```diff
@@ -137,13 +137,17 @@ SR_API GSList *sr_buildinfo_libs_get(void)
 		glib_binary_age, glib_interface_age));
 	l = g_slist_append(l, m);
 
+#ifdef CONF_ZLIB_VERSION
 	m = g_slist_append(NULL, g_strdup("zlib"));
 	m = g_slist_append(m, g_strdup_printf("%s", CONF_ZLIB_VERSION));
 	l = g_slist_append(l, m);
+#endif
 
+#ifdef CONF_LIBZIP_VERSION
 	m = g_slist_append(NULL, g_strdup("libzip"));
 	m = g_slist_append(m, g_strdup_printf("%s", CONF_LIBZIP_VERSION));
 	l = g_slist_append(l, m);
+#endif
```

Comment:
In fact libzip is a required dependency, libsigrok will not build without it. So there is no need for this #ifdef.

Reply:
OK will do. Actually funny story, I am so dependent on CLion for my dev work that, in order to develop this patch, I reimplemented about half the libsigrok build system in CMake so that I could open the project in CLion. So that's how I found this, but I am happy to revert it.

btw, if you guys are interested in a CMake build system option, I could see about polishing up that code some more and contributing it (though I am not 100% sure I still have it anymore)